src/api/job.js

```diff
@@ -0,0 +1,56 @@
+const {Router} = require('express');
+const Job = require('../persistence/job');
+
+const router = new Router();
+
+// Admin Create a new job with post method
+router.post('/', async (request, response) => {
+  try {
+    const {
+      title,
+      salaryrange,
+      description,
+      tags,
+      company,
+      logoURL
+    } = request.body;
+    if (
+      !title ||
+      !salaryrange ||
+      !description ||
+      !tags ||
+      !company ||
+      !logoURL
+    ) {
```

Comment:
The condition format is hard to follow if we break it into multiple lines. Should we format it to one line instead?

Reply:
This one is the result of the format "xo prettier" when I run the unit test. It doesn't allow write those conditions in one line. I'm trying to figure out why.

Reply:
Ah okey. If it's the default settings of `xo prettier`, we can use it.